### PR TITLE
Model Library detection has changed on August 2nd

### PR DESCRIPTION
**Pull request by julien-c**

### CODE REVIEW

**Review by SBrandeis — APPROVED** [review]

**Review by pcuenca — APPROVED** [review]
Oh, St. Julien came! 😂

**Review by Vaibhavs10 — APPROVED** [review]
The day has finally come! 🔥 

Obligatory meme:
image

**Review by osanseviero — APPROVED** [review]
Thanks!

File: docs/hub/model-cards.md [hunk 1]
```diff
@@ -98,10 +98,10 @@ tags:
 - flair
 ```
 
-If it's not specified, the Hub will try to automatically detect the library type. Unless your model is from `transformers`, this approach is discouraged and repo creators should use the explicit `library_name` as much as possible. 
+If it's not specified, the Hub will try to automatically detect the library type. However, this approach is discouraged and repo creators should use the explicit `library_name` as much as possible.
```
Comment (osanseviero) [inline docs/hub/model-cards.md:101]:
```suggestion
If it's not specified, the Hub will try to automatically detect the library type. However, this approach is discouraged, and repo creators should use the explicit `library_name` as much as possible. 
```

**Comment by julien-c** [comment]
thx for merging @osanseviero

**Pull request merged**